### ISSUE TITLE
Possibility to disable saving the history.

### ISSUE THREAD

**Issue by mtstv** [Netcentric/accesscontroltool#143]
I would like to disable history. Would modify the file
/accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/installationhistory/impl/AcHistoryServiceImpl.java

```
    @Override
    public void persistHistory(AcInstallationHistoryPojo history) {

        if (nrOfSavedHistories == 0) {
        	String message = "History hasn't been persisted, configured number of histories is " +nrOfSavedHistories;
            history.addVerboseMessage(message);
            LOG.info(message);
            return;
        }
```

**Comment by gruberroland-netcentric** [comment]
Makes sense, can you create a pull request?

**Comment by mtstv** [comment]
Already done.
https://github.com/Netcentric/accesscontroltool/pull/144

**Comment by gruberroland-netcentric** [comment]
Thanks! :)

**Comment by ghenzler** [comment]
@mtstv merged #144, feel free to create a local intermediate release to solve the urgent issues are your current client project.
